src/app/video-courses/course-item/course-item.component.ts

```diff
@@ -1,15 +1,21 @@
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 
 @Component({
   selector: 'app-course-item',
   templateUrl: './course-item.component.html',
   styleUrls: ['./course-item.component.css']
 })
-export class CourseItemComponent {
+export class CourseItemComponent implements OnInit {
   @Input() courseId: number;
+
   @Input() courseTitle: string;
   @Input() courseDescription: string;
-  @Input() courseDate: Date;
+  @Input() courseDateMillis: number;
+  courseDate: Date;
   @Input() courseDuration: number;
   @Output() courseDeleted = new EventEmitter<number>();
+
+  ngOnInit(): void {
+    this.courseDate = new Date(this.courseDateMillis);
```

Comment:
Better to transform timestamp to `Date` object inside service. If you want to keep it in component better to transform in property setter instead.

Reply:
Moved to service.